frontend/pages/software/components/EmptySoftware.tsx

```diff
@@ -1,3 +1,5 @@
+// This component is used on ManageSoftwarePage.tsx and Homepage.tsx > Software.tsx card
```

Comment:
Should we organize this component into another directory? I have it used outside of pages/software for the pages/Homepage/cards/software

Reply:
I'm in favor of moving this. @RachelElysia, where do you think it makes the most sense for us to put components like this one?

Reply:
https://github.com/fleetdm/fleet/issues/4966